Dockerfile

```diff
@@ -6,4 +6,5 @@ COPY . /app
 RUN pip install -r requirements.txt
 
 RUN chmod +x src/main.py
-CMD python -u src/main.py
+CMD sleep infinity
```

Comment:
why do we need this cmd (`sleep infinity`)?

Reply:
If there is no command at all, the container will start and shut down immediately.

Reply:
alright, makes perfect sense

Reply:
Didn't work on my side, needed to add `ENTRYPOINT` below